pkg/scheduler/framework/runtime/framework.go

```diff
@@ -335,10 +329,7 @@ func NewFramework(r Registry, profile *config.KubeSchedulerProfile, opts ...Opti
 			continue
 		}
 
-		args, err := getPluginArgsOrDefault(options.componentConfigVersion, pluginConfig, name)
-		if err != nil {
-			return nil, fmt.Errorf("getting args for Plugin %q: %w", name, err)
-		}
+		args := pluginConfig[name]
```

Comment:
what about cluster autoscaler?

Reply:
Yes, I have it on my todo list, they need to create a versioned CC, run it through the defaulting logic to get an internal type and then pass that when instantiating the framework.